src/index.html

```diff
@@ -11,7 +11,143 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="field-input">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          minlength="2"
+          maxlength="15"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            minlength="2"
            maxlength="15"
            autocomplete="off"
          >
```

fix indentations in the document
checklist points 2 and 3